AppAuth.podspec

```diff
@@ -33,16 +33,21 @@ It follows the OAuth 2.0 for Native Apps best current practice
   s.platforms    = { :ios => "7.0", :osx => "10.9", :watchos => "2.0", :tvos => "9.0" }
 
   s.source       = { :git => "https://github.com/openid/AppAuth-iOS.git", :tag => s.version }
-
-  s.source_files = "Source/*.{h,m}"
   s.requires_arc = true
 
-  # iOS
-  s.ios.source_files      = "Source/iOS/**/*.{h,m}"
-  s.ios.deployment_target = "7.0"
-  s.ios.frameworks        = "SafariServices", "AuthenticationServices"
-
-  # macOS
-  s.osx.source_files = "Source/macOS/**/*.{h,m}"
-  s.osx.deployment_target = '10.9'
+  s.subspec 'Core' do |core|
+     # subspec for users who don't want the authentication code
+     core.source_files = "Source/*.{h,m}"
```

Comment:
Can this line be moved outside the subspec?

Reply:
the reason why I had to put it inside the subspec is the following cocoapods error:

```
[!] The `AppAuth` pod failed to validate due to 1 error:
    - ERROR | File Patterns: The AppAuth/Core (0.94.0) spec is empty (no source files, resources, resource_bundles, preserve paths, vendored_libraries, vendored_frameworks, dependencies, nor subspecs).
```

so as long as the subspec remains empty cocoapods refuses to build the pod...

Reply:
`AppAuth/Core` will work, because that's the subspec which includes all the files in `Source/*.{h,m}` ;-)

But those who include `AppAuth/ExternalUserAgent` explicitly will also work, because the `ExternalUserAgent` subspec has specified `AppAuth/Core` as its dependency

Reply:
Ah ok, sounds good.